apic.go

```diff
@@ -55,12 +55,12 @@ func yaml_parser_initialize(parser *yamlParser) bool {
 }
 
 // Destroy a parser object.
-func yaml_parser_delete(parser *yamlParser) {
+func (parser *yamlParser) delete() {
 	*parser = yamlParser{}
 }
 
 // String read handler.
-func yaml_string_read_handler(parser *yamlParser, buffer []byte) (n int, err error) {
+func yamlStringReadHandler(parser *yamlParser, buffer []byte) (n int, err error) {
```

Comment:
This name was strange and is still strange.
Wording thing is never fun.

But I feel like this method that accepts a yamlParser and a []byte should become a method of yamlParser accepting []byte as argument.

It could help to simplify the wording

So it would become readString

```suggestion
func (parser *yamlParser) readString(buffer []byte) (n int, err error) {
```

Reply:
You are right

Reply:
Changing function names is out of scope for this PR.
Please keep it boring.